egs/librispeech/asr/simple_v1/train.py

```diff
@@ -29,7 +29,9 @@
 from snowfall.common import setup_logger
 from snowfall.models import AcousticModel
 from snowfall.models.tdnn_lstm import TdnnLstm1b
-from snowfall.training.ctc_graph import CtcTrainingGraphCompiler
+from snowfall.training.asg_graph import get_phone_symbols
+from snowfall.training.asg_graph import create_bigram_phone_lm
+from snowfall.training.asg_graph import AsgTrainingGraphCompiler
```

Comment:
Just a question: wondering why we override `train.py` instead of creating a new script for ASG? (or add some config), I was thinking we may keep CTC scripts for some future comparison.

Reply:
You're right. I thought it is faster to reuse the existing script. Will revert changes and create separate scripts for ASG.

Reply:
I think we should have an option to choose between CTC and ASG loss in the same script. Although currently it seems difficult to abstract that away in a clean way, so maybe discard that remark...